packages/atlas/package.json

```diff
@@ -15,13 +15,14 @@
     "tokens:fetch": "node ./scripts/fetchTokens && prettier --write src/styles/tokens",
     "tokens:build": "style-dictionary build --config ./sd.config.js && prettier --write src/styles/generated/variables.ts",
     "tokens": "yarn tokens:fetch && yarn tokens:build",
-    "svgs": "yarn svgs:icons && yarn svgs:illustrations",
-    "svgs:illustrations-generate": "svgr --config-file svgr.config.js -d src/components/_illustrations src/components/_illustrations/svgs && prettier --write src/components/_illustrations",
+    "svgs": "yarn svgs:icons && yarn svgs:illustrations && yarn svgs:logos",
+    "svgs:illustrations-generate": "svgr --config-file svgr.config.js -d src/assets/illustrations src/assets/illustrations/svgs && prettier --write src/assets/illustrations",
     "svgs:illustrations-import": "node ./scripts/figma-import --illustrations",
     "svgs:illustrations": "yarn svgs:illustrations-import && yarn svgs:illustrations-generate",
     "svgs:icons-import": "node ./scripts/figma-import --icons",
-    "svgs:icons-generate": "svgr --config-file svgr.config.js -d src/components/_icons src/components/_icons/svgs && prettier --write src/components/_icons",
+    "svgs:icons-generate": "svgr --config-file svgr.config.js -d src/assets/icons src/assets/icons/svgs && prettier --write src/assets/icons",
     "svgs:icons": "yarn svgs:icons-import && yarn svgs:icons-generate",
+    "svgs:logos": "svgr --config-file svgr.config.js -d src/assets/logos src/assets/logos/svgs && prettier --write src/assets/logos",
```

Comment:
Let's add it to the global package.json. WDYT?
`"atlas:svgs:logos": "yarn workspace @joystream/atlas svgs:logos",`

Reply:
Done